tests/modules/individuals/test_models.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+
+
+def test_individual_add_remove_encounters(encounter_1, encounter_2, empty_individual):
+
+    assert len(empty_individual.encounters) == 0
```

Comment:
Likewise, as soon as this test finishes, the shared empty_individual is no longer empty, so any later test that assumes it is will fail

Reply:
Once again did not get Mock to work but restored the test individual to it's original state